test/libsolidity/Metadata.cpp

```diff
@@ -43,7 +43,9 @@ BOOST_AUTO_TEST_CASE(metadata_stamp)
 		}
 	)";
 	CompilerStack compilerStack;
-	BOOST_REQUIRE(compilerStack.compile(std::string(sourceCode)));
+	compilerStack.addSource("", std::string(sourceCode));
+	/// NOTE: compiles without optimisations
```

Comment:
I guess turning optimizations on/off is not the point of this PR, but the comment is a good one.